atomic_reactor/util.py

```diff
@@ -219,7 +219,7 @@ def commit_id(self):
     def git_path(self):
         if self._git_path is None:
             repo_data = clone_git_repo(self.git_url, self._tmpdir, self.commit,
-                                       branch=self._branch, depth=self._git_depth)
+                                       branch=self._branch, depth=self._git_depth, exists_ok=True)
```

Comment:
Could we also change the Source and/or GitSource classes to make sure that nothing except for the `get()` method tries to do the cloning? It's currently hard to be sure that we're actually using the output of the clone task, because the `source` object may decided to clone at any point if you just access the `.path`

Reply:
Changed the source to only clone when get it called directly for GitSource.